config/routes.rb

```diff
@@ -34,9 +34,26 @@
   get '/orders/new', to: 'orders#new'
   post '/orders', to: 'orders#create'
   get '/orders/:id', to: 'orders#show'
+
   #register
   get '/register', to: 'register#new'
   post '/register', to: 'register#create'
+
+  #login
+  get '/login', to: 'sessions#new'
+  post '/login', to: 'sessions#create'
+
+  #logout
+  get '/logout', to: 'sessions#destroy'
+
   #profile
   get '/profile', to: 'profile#show'
+
+  namespace :admin do
+    get '/', to: "dashboard#index"
+  end
+
+  namespace :merchant do
+    get '/', to: "dashboard#index"
+  end
```

Comment:
We should change the controller action here from index to show because the dashboard is a singleton